pkgs/applications/networking/appgate-sdp/default.nix

```
@@ -92,11 +96,11 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "appgate-sdp";
-  version = "5.1.2";
+  version = "5.3.2";
 
   src = fetchurl {
-    url = "https://bin.appgate-sdp.com/5.1/client/appgate-sdp_${version}_amd64.deb";
-    sha256 = "0v4vfibg1giml3vfz2w7qypqzymvfchi5qm6vfagah2vfbkw7xc2";
+    url = "https://bin.appgate-sdp.com/${version:0:3}/client/appgate-sdp_${version}_amd64.deb";
```

This won't work, it we don't do substrings like this. You probably want the following:
```suggestion
    url = "https://bin.appgate-sdp.com/${lib.versions.majorMinor version}/client/appgate-sdp_${version}_amd64.deb";
```